README.md

```diff
@@ -50,7 +55,12 @@ To configure the module behaviour, provide a non-empty JSON object to the `--exe
             // - nimbus: statusim/nimbus-eth2:amd64-latest",
             // - prysm: prysmaticlabs/prysm-beacon-chain:latest,prysmaticlabs/prysm-validator:latest",
             // - lodestar: chainsafe/lodestar:next",
-            "clImage": ""
+            "clImage": "",
+
+            // The log level that the CL client should log at, it overrides the value set in the `logLevel` property
```

Comment:
Recommend using the same suggested text as above, and just replace the EL-specific stuff with CL-specific stuff (though watch out, you have a typo of "Teky" here)